source/Ship.cpp

```diff
@@ -2853,6 +2854,10 @@ bool Ship::IsReadyToJump(bool waitingIsReady) const
 			|| hyperspaceCount || !targetSystem || !currentSystem)
 		return false;
 
+	cout<<"Hi: "<<position.Length()<<" "<<currentSystem->DepartureDistance()<<endl;
+	if(position.Length() <= currentSystem->DepartureDistance())
+		return false;
+
```

Comment:
I think you forgot to change this since your tests?
we definitely should have a warning message for the player saying "moving to the closest jumping point" or smth of the like